src/drive/drive.ts

```diff
@@ -874,7 +874,8 @@ function downloadResource(resource: FilesResource, picked: boolean = false): Pro
       // it with a picker to explicitly grant permission.
       if(error.xhr.responseText === 'appNotAuthorizedToFile'
          && picked === false) {
-        return pickFile(resource).then(() => {
+        //TODO: get the client Id here properly.
```

Comment:
Yes, this is a minor regression. `pickFile` is not used with the current permissions settings, but if it were to be used, this would be broken :smile: